src/Resources/app/administration/src/module/sw-mail-template/component/sw-mail-template-list/index.js

```diff
@@ -0,0 +1,37 @@
+import template from './sw-mail-template-list.html.twig';
+
+Shopware.Component.override('sw-mail-template-list', {
+    template,
+    methods: {
+        getListColumns() {
```

Comment:
We should call the parent method to support plugins

Reply:
for sure. I just forgot that.